controllers/managedocs_controller.go

```diff
@@ -1058,6 +1079,95 @@ func (r *ManagedOCSReconciler) reconcilePrometheus() error {
 	return nil
 }
 
+func (r *ManagedOCSReconciler) reconcilePrometheusRemoteWrite() (*promv1.RemoteWriteSpec, error) {
```

Comment:
```suggestion
func (r *ManagedOCSReconciler) getPrometheusRemoteWrite() (*promv1.RemoteWriteSpec, error) {
```
- I don't see this method doing any reconcile, so imho `get...` in method signature is more apt

Reply:
Done